.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 'iojs-v1'
```

Comment:
Not a comment about the code, but it feels weird that this is how Travis decided to version io.js